src/chains/algorand_1/examples/transfers.ts

```diff
@@ -38,12 +38,23 @@ const composeTokenTransferParams: Partial<AlgorandActionAssetTransferParams> = {
 }
 
 const composeAlgoPaymentParams: Partial<AlgorandActionPaymentParams> = {
```

Comment:
Rekey is a special feature. Can you move the example out of transfers and into its own accountRekey.ts file?